framecrypt.py

```
@@ -0,0 +1,215 @@
+
+    start_music("decode", args.music)
+    out_dir = args.dir
+    with tempfile.TemporaryDirectory() as tmp:
+        zip_path = os.path.join(tmp, "decoded.zip")
+        fc_video.decode(video, zip_path, block=args.block)
+        for attempt in range(3):
+            try:
+                fc_crypto.extract(zip_path, out_dir, getpass.getpass("Password to extract: "))
+                print(f"✓ extracted to {out_dir}/")
+                return
+            except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)